biochatter/vectorstore_agent.py

```diff
@@ -625,3 +625,19 @@ def get_all_documents(
         except MilvusException as e:
             logger.error(e)
             raise e
+        
+    def get_description(self, doc_ids: Optional[List[str]]=None):
+        def get_name(meta: Dict[str, str]):
+            name_col = ["title", "name", "subject", "source"]
+            for col in name_col:
+                if meta[col] is not None and len(meta[col]) > 0:
+                    return meta[col]
+            return ""
+
+        expr = VectorDatabaseAgentMilvus._build_meta_col_query_expr_for_all_documents(doc_ids)
+        result = self._col_metadata.query(
+            expr=expr, output_fields=METADATA_FIELDS,
+        )
+        names = list(map(get_name, result))
+        names_set = set(names)
+        return f"This vector store contains the following articles: {names_set}"
```

Comment:
Do we need to limit this for large vector stores? Should we give an option to manually describe a vector store, going beyond the names of articles in it?

Reply:
It's a good point, I think we can enable to add descriptions in the customization yaml file and pass them to RagAgent

Reply:
@slobentanzer  I added RagAgent.agent_desc, which will override description of DatabaseAgent, VectorstroeAgent and API agent if it is set